Monika-After-Story/MonikaModDev
# Timetravel hardstop event

Since people keep messing with the time while knowing it will break the mod, we need to add a hard stop in case the time is incorrect. This way they can at least restore a backup before it's too late. Also makes sense for Monika to react on what's happening.

### Changes:
- added the new event
- new system for handling tt
- kazoo is now officially in and is used

### Testing:
- verify there's no false positives with time travelling
- try to break the time and verify you get the hard stop event
- verify you can't load in afterwards
- verify the expressions and writing are good (not that we care, we do **not** want people to actually see this event)
